types/mapbox-gl/v1/tslint.json

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "array-type": false,
+        "ban-types": false,
+        "export-just-namespace": false,
+        "interface-name": false,
+        "interface-over-type-literal": false,
+        "jsdoc-format": false,
+        "max-line-length": false,
+        "no-any-union": false,
+        "no-empty-interface": false,
+        "no-redundant-jsdoc-2": false,
+        "no-unnecessary-generics": false,
+        "no-unnecessary-qualifier": false,
+        "no-var-keyword": false,
+        "object-literal-shorthand": false,
+        "only-arrow-functions": false,
+        "prefer-const": false,
+        "space-before-function-paren": false,
+        "strict-export-declare-modifiers": false,
+        "triple-equals": false
+    }
```

Comment:
Stick to the required tslint form

Reply:
@elibarzilay 

Thank you for the review! Does "required tslint form" mean below in README?

> The linter configuration file, tslint.json should contain { "extends": "dtslint/dt.json" }, and no additional rules.

If the rules are disabled, so many errors occurs. 

The purpose of this PR is just to want to fix a minor issue that affects mapbox GL JS v1.13.x function (it's a maintenance phase version in Mapbox) . Therefore, I want to keep almost all as-is.